AutoRouteManager.php

```diff
@@ -64,20 +71,43 @@ public function __construct(
      */
     public function buildUriContextCollection(UriContextCollection $uriContextCollection)
     {
-        $this->getUriContextsForDocument($uriContextCollection);
+        $this->collectionBuilder->build($uriContextCollection);
 
         foreach ($uriContextCollection->getUriContexts() as $uriContext) {
-            $existingRoute = $this->adapter->findRouteForUri($uriContext->getUri(), $uriContext);
+            $translatedSubject = null;
```

Comment:
this is not needed, as it's redefined one line 81 and only used in that if statement.